config/project/project.yaml

```diff
@@ -111,6 +111,19 @@ elementSources:
         - dateUpdated
         - revisionCreator
       type: native
+    -
+      defaultSort:
+        - 'fieldInstance:e30e5ed4-2a93-4299-927a-7993d8dc68b3'
+        - desc
+      defaultViewMode: ''
+      disabled: false
+      key: 'section:a976662e-fc19-4f89-ae6b-a5b4db64752b' # Events
+      tableAttributes:
+        - link
+        - 'fieldInstance:04ba682b-6348-48de-833f-c03c6dd10b6b'
```

Comment:
Hide the poster field from this index, make it a part of Event entry type’s _Element Thumbnail_.